src/features/files/components/FileLibraryDialog/FilePreview.tsx

```diff
@@ -14,31 +14,45 @@ type Props = {
 };
 
 const FilePreview: FC<Props> = ({ file, onBack, onSelect }) => {
+  const [dimensions, setDimensions] = useState({ height: 0, width: 0 });
+
   return (
     <Box display="flex" flexDirection="column" height="100%">
       <Box height="100%" my={2} overflow="auto">
         <TransparentGridBackground
           interactive={false}
           sx={{
-            height: 'calc(100% - 2em)',
+            height: 'calc(100% - 4em)',
             position: 'relative',
           }}
         >
           <Image
             alt={file.original_name}
             height="800"
+            onLoad={(e) => {
+              if (e.target instanceof HTMLImageElement) {
+                setDimensions({
+                  height: e.target.naturalHeight,
+                  width: e.target.naturalWidth,
+                });
+              }
+            }}
             src={file.url}
             style={{
               height: '100%',
               objectFit: 'contain',
               width: '100%',
             }}
+            unoptimized
             width="800"
           />
         </TransparentGridBackground>
         <Typography color="secondary" mt={1} textAlign="center" variant="body2">
           {file.original_name}
         </Typography>
+        <Typography color="secondary" mt={1} textAlign="center" variant="body2">
+          {dimensions.width} &times; {dimensions.height} pixels
```

Comment:
With the change to "pixels", this is becoming a string that I believe requires localizing. Would you mind creating a message in `messageIds.ts` of the `files` feature, and use it here, with `height` and `width` as parameters?

Reply:
Oops forgot that!